package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "create-nightwatch",
-  "version": "1.5.3",
+  "version": "4",
```

Comment:
why 4?

Reply:
I don't know... I had it 3 and then it was suddenly 4. fixed it